jupyter_server/pytest_plugin.py

```diff
@@ -498,3 +499,8 @@ async def _():
                 print(e)
 
     return _
+
+
+def authorize_all():
+    yield
+    JupyterHandler.is_authorized = lambda self, user, action, resource: True
```

Comment:
This method (partially implemented fixture) also seems redundant?

Reply:
Good catch. Removed.